codec-registration-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/ClassSweepCompilerPluginComponent.scala

```diff
@@ -20,7 +20,7 @@ class ClassSweepCompilerPluginComponent(options: CodecRegistrationCheckerOptions
 
   override def newPhase(prev: Phase): Phase =
     new StdPhase(prev) {
-      private val up = options.oldTypes.groupBy(_._2)
+      private val oldTypesFromPreviousCompilation = options.oldTypes.groupBy(_._2)
 
       override def apply(unit: global.CompilationUnit): Unit = {
         val body = unit.body
```

Comment:
While. you're here... I'd also rename `x`es from below into sth more meaningful (esp. that `x` seems to correspond to both `ClassDef`s and the type of their `impl`s

Reply:
Plus decompose tuple explicitly in line 33, to avoid `_1` and `_2` which aren't particularly clear

Reply:
Done - please take a look